packages/react-icons/src/createIcon.tsx

```diff
@@ -62,20 +62,16 @@ export function createIcon({
     id = `icon-title-${currentId++}`;
 
     render() {
-      const { size, color, title, noVerticalAlign, ...props } = this.props;
+      const { size, color, title, noVerticalAlign, className, ...props } = this.props;
 
       const hasTitle = Boolean(title);
-      const heightWidth = getSize(size);
-      const baseAlign = -0.125 * Number.parseFloat(heightWidth);
-      const style = noVerticalAlign ? null : { verticalAlign: `${baseAlign}em` };
       const viewBox = [xOffset, yOffset, width, height].join(' ');
+      const classes: String[] = [className, noVerticalAlign ? '' : getSizeClass(size)].filter(t => t);
```

Comment:
Can you explain like I'm 5 what this does? 🙃 Just want to make sure it allows you to set a size class and `noVerticalAlign` independently (both set or not, or either set and the other not)

Reply:
Ahh I see - it looks like this logic was not setting a size class if `noVerticalAlign` was set. Just pushed a fix for this